### PR TITLE
chore: Implement locators for automation on the new navigation (WPB-9506)

### PR DESCRIPTION
**Pull request by V-Gira**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-9506" title="WPB-9506" target="_blank">TaskWPB-9506</a>  [Web] Implement locators for automation on the new navigation
  </td></table>
  <br />
 

## Description

Adding and changing a few locators for automation according to this feedback from QA:

> - Use the same locator for the conversation list in every view
> - Add data-uie-name to element with class ‘left-list-header’ -> “conversation-list-header”
> - Add data-uie-name to element with class class ‘css-x122mf’ -> “conversation-list-header-title”
> - Add data-uie-name to the input element that searches vor conversations -> “search-conversations”
> - Add data-uie-name to element with class ‘conversations-sidebar-btn--badge’ -> “unread-badge”
> - Add data-uie-name to element with class 'conversations-sidebar-folders--inner-wrapper' -> "folder-list"

### CODE REVIEW

File: src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx [hunk 1]
```diff
@@ -75,9 +75,9 @@ export const ConversationHeader = ({
   return (
     <>
       <div css={header}>
-        <span css={label}>
+        <h2 css={label} data-uie-name="conversation-list-header-title">
```
Comment (V-Gira) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx:78]: changing `span` to `h2` for a11y

Comment (przemvs) [reply src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx:78]: There is no added additional margins from default h2 styles? 🤔

File: src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx [hunk 2]
```diff
@@ -155,13 +155,11 @@ export const ConversationsList = ({
...
 
-  const uieName = isFolderView ? 'folder-view' : 'recent-view';
-
   return (
     <>
-      <h2 className="visually-hidden">{t(isFolderView ? 'folderViewTooltip' : 'conversationViewTooltip')}</h2>
+      <h2 className="visually-hidden">{t('conversationViewTooltip')}</h2>
 
-      <ul css={css({margin: 0, paddingLeft: 0})} data-uie-name={uieName}>
+      <ul css={css({margin: 0, paddingLeft: 0})} data-uie-name={'conversation-view'}>
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx]:
```suggestion
      <ul css={css({margin: 0, paddingLeft: 0})} data-uie-name="conversation-view">
```

File: src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx (hunk 1 above)
Comment (V-Gira) [reply src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx:78]: I'll double check

Comment (V-Gira) [reply src/script/page/LeftSidebar/panels/Conversations/ConversationHeader/ConversationHeader.tsx:78]: we remove those user agent margins in `normalize.less`

**Review by przemvs — APPROVED** [review]

**Pull request merged**